### ISSUE TITLE
Dim the screen backlight / avoid blinking the 7 segment

### ISSUE THREAD

**Issue by X-Ryl669** [ObooSmartClock/Oboo-Packages#1]
Currently, for an alarm clock, Oboo SmartClock is too bright IMHO. Also, it's blinking and it's a real pain when you're falling asleep.

I've found a way to handle the brightness of the LCD screen:
```
echo 0 > /sys/class/pwm/pwmchip0/export
echo 10000000 > /sys/class/pwm/pwmchip0/pwm0/period
echo  9899999 > /sys/class/pwm/pwmchip0/pwm0/duty_cycle
echo 1 > /sys/class/pwm/pwmchip0/pwm0/enable
```
The duty cycle could go from 9999999 (black) to 0 (bright), but 8999999 is already very bright.

For the blinking issue, I've found where it's happening and I need help fixing it.

**Comment by X-Ryl669** [comment]
Ok, in clock-mcu package, this code:
```
     if (millis() - prevSeparatorTime >= separatorInterval) {
		flipTimeDisplaySeparator();

		// prepare for next interval
		prevSeparatorTime = millis();
	}
```
should be optional (or at least, conditional to being run only during the day).
Would you accept a pull request to modify it ?

**Comment by jjok** [comment]
Yeah, definitely far too bright. I haven't noticed the blinking, though.

**Comment by greenbreakfast** [comment]
Info display backlight:
An easier way to control it is with the `onion pwm` command.
```
# set GPIO18 to PWM
omega2-ctrl gpiomux set uart1 pwm01
# very dim
onion pwm 0 90 60
# bright
onion pwm 0 10 60
```

For more info on the `onion pwm` command, try running `onion --help`

---
Blinking colon: Yes, will definitely look into your PR.

---
For future reference, it's much easier to keep track of everything if you keep it to one topic per GitHub issue.
